app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    review = Review.find_by(id: params[:review_id])
+    like = Like.new(review: review, user: current_user)
+    if like.save && request.xhr?
```

Comment:
It would be better to check the ajaxy-ness and success of the save separately as you may wish to branch differently in either case.